### PR TITLE
[sparkle] - refacto: deprecate old DropdownMenu

### PR DESCRIPTION
**Pull request by JulesBelveze**
## Description

This PR deprecates the old `DropdownMenu` component. To do so it refactors `Breadcrumbs` and `DataTable` to use the `NewDropdownMenu` component instead.

Note: `DropdownMenu` is no longer used in `front`

## Risk

UI breaking changes

## Deploy Plan

- Publish sparkle

### CODE REVIEW

File: sparkle/src/components/Breadcrumbs.tsx [hunk 1]
```diff
@@ -56,24 +63,24 @@ export function Breadcrumbs({ items }: BreadcrumbProps) {
...
-                    </DropdownMenu.Item>
-                  ))}
-                </DropdownMenu.Items>
-              </DropdownMenu>
+              <NewDropdownMenu>
+                <NewDropdownMenuTrigger asChild>
+                  <Button variant="ghost" label={ELLIPSIS_STRING} />
+                </NewDropdownMenuTrigger>
+
+                <NewDropdownMenuContent align="start" className="min-w-[220px]">
```
Comment (flvndvd) [inline sparkle/src/components/Breadcrumbs.tsx:71]: Do we really need the arbitrary value, here?

File: sparkle/src/components/DataTable.tsx [hunk 2]
```diff
@@ -353,21 +357,24 @@ DataTable.Row = function Row({
...
-            </DropdownMenu.Button>
-            <DropdownMenu.Items origin="topRight" width={220}>
-              {moreMenuItems?.map((item, index) => (
-                <DropdownMenu.Item key={index} {...item} />
-              ))}
-            </DropdownMenu.Items>
-          </DropdownMenu>
+            </NewDropdownMenuTrigger>
+
+            <NewDropdownMenuContent align="end" className="w-[220px]">
```
Comment (flvndvd) [inline sparkle/src/components/DataTable.tsx:370]: Same

File: sparkle/src/stories/Item.stories.tsx [hunk 3]
```diff
@@ -79,61 +84,74 @@ export const ListItemExample = () => (
     <div className="s-flex s-flex-col s-gap-8">
```
Comment (flvndvd) [inline sparkle/src/stories/Item.stories.tsx:84]: Why not killing this file, it's superseeded by `NewDropdown.stories.tsx`

**Review by flvndvd — APPROVED** [review]

File: sparkle/src/components/Breadcrumbs.tsx (hunk 1 above)
Comment (JulesBelveze) [reply sparkle/src/components/Breadcrumbs.tsx]: Not really 👍🏼

File: sparkle/src/stories/Item.stories.tsx (hunk 3 above)
Comment (JulesBelveze) [reply sparkle/src/stories/Item.stories.tsx:84]: Well we still have examples of how to use Item, Item.Avatar, Item.Link and stuff

Comment (flvndvd) [reply sparkle/src/stories/Item.stories.tsx]: Gracias!

**Pull request merged**